### PR TITLE
:lipstick: Make it look like it is from the 90s

### PR DESCRIPTION
**Pull request by whitep4nth3r**
Rather than custom styles mentioned in [this issue](https://github.com/whitep4nth3r/the-claw-webring-widget/issues/3), how about we make the component look absolutely outrageous so it doesn't fit with _anyone's_ branding?

### CODE REVIEW

**Comment by mdeloughry** [comment]
Agreed embrace the wackiness that is the claw webring!!!

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
       <h1>The Claw Webring Widget</h1>
       <script type="module" src="src/index.js"></script>
 
-      <the-claw-webring-widget hideMembers="false">
+      <the-claw-webring-widget hideMembers="false" theme="dark">
```
Comment (endymion1818) [inline index.html:14]: Does this matter?

**Review by rknightuk — APPROVED** [review]
gimmmeeeeee

File: src/main.css [hunk 2]
```diff
@@ -8,130 +8,185 @@
...
 }
 
 :host([theme="light"]) {
   --color-bg: var(--white);
   --color-fg: var(--black);
+
```
Comment (endymion1818) [inline src/main.css:32]: Extra line here

**Review by endymion1818 — APPROVED** [review]

**Pull request merged**